readme.md

```diff
@@ -82,264 +101,296 @@ var civic = {
+```bash
+$ touch index.html script.js  # create html and js files
+$ open index.html # open index.html in the browser so you can test your code in the console
+$ atom . # open the files in Atom and start coding
+```
 
-Creating all these copies can cause our program to use more memory than it
-really needs to.
+For this exercise you will be creating an ATM class.
 
-Thankfully, by using **prototypes**, we can eliminate this duplication. Every
-object in JS has a `prototype` property, that points to another **object**.
-Almost always, the prototype of a given object will come from it's
-**constructor** (which has a prototype).
+It will have the following properties...
+* `type` (e.g., "checking"), which should be determined by some input
+* `money`, which should start out as `0`
 
-Any properties / methods defined on an object's prototype are available on the
-that object. An example:
+It should have the following methods...
+* `withdraw`, which should increase the amount of money by some input
+* `deposit`, which should decrease the amount of money by some input
+* `showBalance`, which should print the amount of money in the bank to the console.
 
-```js
-function Dog(name, breed) {
-  this.name = name;
-  this.breed = breed;
-}
+The `Atm` class a `transactionHistory` property which keeps track of the withdrawals and deposits made to the account.
+* Make sure to indicate whether the transaction increased or decreased the amount of money in the bank.
 
-Dog.prototype.species = "Canis Canis";
-Dog.prototype.bark = function() { return "Woof! I'm " + this.name; }
+#### Bonus
 
-// OR Alternate form:
-// The disadvantage here is that we're overwriting any existing properties on
-// the prototype
-Dog.prototype = {
-  species: "Canis Canis",
-  speak: function() { return "Woof! I'm " + this.name; }
-}
-
-// Our objects work just as they did before!
-var spot = new Dog("Spot", "Beagle");
-var rufus =  new Dog("Rufus", "Poodle");
+Give the `Atm` class a `backupAccount` property that can, optionally, contain a reference to another instance of the class, determined by some input
+* Whenever an ATM's balance goes below zero, it will remove money from the instance stored in `backupAccount` so that its balance goes back to zero.
+* This should trigger a withdrawal in the back up account equal to the amount of money that was withdrawn from the original account.
 
-spot.name // "Spot"
-spot.breed // "Beagle"
-spot.bark() // "Hello! I'm Spot"
-
-rufus.name // "Rufus"
-rufus.bark() // "Hello! I'm Rufus"
-```
+<!-- AM: Add solution. -->
 
-![Prototype Chain Diagram - Simple](images/prototype_chain_simple.jpg)
+## Break (10 minutes / 1:15)
 
-**Note**: `.prototype` is a property on constructor functions, while `.__proto__` is the
-property on objects, and is used in the lookup chain.
+## Inheritance (15 minutes / 1:30)
```

Comment:
It might be worth talking about the relationship between cat, dog, and animal here before looking at any code. Maybe a diagram or asking the students what they have in common